i18n/zh/docusaurus-plugin-content-docs/current/instruction/03-runtime.md

```diff
@@ -1,216 +1,271 @@
-# EventMesh Runtime 快速入门说明
+# EventMesh Runtime 快速开始
 
 EventMesh Runtime 是 EventMesh 集群中有状态的 Mesh 节点，负责 Source Connector 与 Sink Connector 之间的事件传输，并可以使用 Event Store 作为事件的存储队列。
 
-![EventMesh Runtime](/images/design-document/runtime.png)
+![EventMesh Runtime](../../../../../static/images/design-document/runtime.png)
```

Comment:
Isn't using the writing method of `/static/images/...` simpler?

Reply:
Using `/static/images/...` will cause docusaurus build error:

```
[INFO] [zh] Creating an optimized production build...


Error: Image static/static/images/design-document/runtime.png used in i18n/zh/docusaurus-plugin-content-docs/current/instruction/03-runtime.md not found.
    at async Promise.all (index 0)
[ERROR] Client bundle compiled with errors therefore further build is impossible.
```